app/models/company.rb

```diff
@@ -1,2 +1,7 @@
 class Company < ActiveRecord::Base
+  validates :name,            presence: true
+  validates :description,     presence: true
+  validates :sub_description, presence: true
```

Comment:
@arafatm Yes `name, title, description` sounds better, Can I change them accordingly and merge?